util/runtests.sh

```diff
@@ -11,6 +15,13 @@
 #  * integtest-full-$DATE.log: results of the integration tests
 #  * ./integtest-$DATE/ :logs of integration testing (2 files per conf)
 
+# Only when the first argument contains pytest the unittests are run using pytests and a pytest report is made.
+if [[ $1 = *pytest* ]]; then
```

Comment:
Let's just write explicitly `if [[ $1 = "--pytest" ]];`
It's clearer what it does.